src/vs/workbench/browser/parts/media/compositepart.css

```diff
@@ -11,6 +11,10 @@
 	display: flex;
 }
 
+.monaco-workbench.mac.use-inline-toolbar > .part > .composite.title {
```

Comment:
This is bad, it is causing me to drag the window when I drag the title of the output panel for example.